docs/CMakeLists.txt

```diff
@@ -13,11 +13,26 @@ set(DOXYGEN_USE_MDFILE_AS_MAINPAGE ${HOMEPAGE_MD})
 set(DOXYGEN_TIMESTAMP YES)
 set(DOXYGEN_WARN_AS_ERROR YES)
 set(DOXYGEN_SOURCE_BROWSER YES)
+set(DOXYGEN_WARN_IF_UNDOCUMENTED NO)
```

Comment:
I guess you're still missing something like
```cmake
set(DOXYGEN_GENERATE_TAGFILE "${CMAKE_CURRENT_BINARY_DIR}/${CMAKE_PROJECT_NAME}.doxygen.tag")
```

Reply:
Seems that you should put the artifact in `${CMAKE_CURRENT_BINARY_DIR}/html` to ensure that our CI picks it up in the pages artifacts :wink:

Reply:
OK, it works.